apps/site/lib/site_web/controllers/schedule/green.ex

```diff
@@ -48,6 +48,10 @@ defmodule SiteWeb.ScheduleController.Green do
   def trip_view(conn, _params) do
     conn
     |> assign(:tab, "trip-view")
+    |> call_plug(SiteWeb.Plugs.BannerMessage,
```

Comment:
Oh I just noticed this one isn't using `call_plug_with_opts`... does this work as desired then?

Reply:
So in this particular file, there is an existing `call_plug` function that I'm calling and has the same code that the one in `SiteWeb.ControllerHelpers`. I didn't want to remove it from this file but it's true that then we have 2 twin functions. I'm rather have just one but let me know if keeping the original one makes more sense?

Reply:
Ohh I see. I agree with you let's have just one - so let's remove this call_plug and update to use the one in SiteWeb.ControllerHelpers.